strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenArtifactControllerTest.java

```diff
@@ -682,27 +689,12 @@ public void whenRepositoryIsOutOfServiceWeShouldDisallowArtifactDeployment()
     }
 
     @Test
-    public void testNonExistingDirectoryDownload()
+    public void testDirectoryDownload()
     {
         String path = "/storages/storage-common-proxies/maven-central/john/doe/";
         ExtractableResponse response = client.getResourceWithResponse(path, "");
-        assertEquals(HttpStatus.NOT_FOUND.value(), response.statusCode(), "Wrong response");
-    }
-
-    @Test
-    public void testNonExistingArtifactDownload()
-    {
-        String path = "/storages/storage-common-proxies/maven-central/john/doe";
-        ExtractableResponse response = client.getResourceWithResponse(path, "");
-        assertEquals(response.statusCode(), HttpStatus.NOT_FOUND.value(), "Wrong response");
-    }
-
-    @Test
-    public void testNonExistingArtifactInNonExistingDirectory()
-    {
-        String path = "/storages/storage-common-proxies/maven-central/john/doe/who.jar";
```

Comment:
we probably should make new test case for not found artifact with valid path.

Reply:
there is already `MavenArtifactControllerTest#testNonExistingArtifactInExistingDirectory` and it is testing is just fine